K2Field.K2NE.ServiceBroker/Helpers/Extensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace K2Field.K2NE.ServiceBroker.Helpers
+{
+    public static class Extensions
+    {
+        public static string ToStringOrEmpty(this Object value)
+        {
+            return value == null ? "" : value.ToString();
```

Comment:
Why not use String.Empty?
Also, i would write the full if statement for readability :)